src/components/ArticlePage.jsx

```diff
@@ -19,6 +20,9 @@ function ArticlePage() {
       <section className="votes">
         <Votes votes={currentArticle.votes} article={article} />
       </section>
+      <section className="comments">
+        <CommentList article={article} />
```

Comment:
```js
<section className="comments">
        <CommentList article={article} />
```
What's the purpose of the `section` here if the `CommentList` component is going to return a `section` as well?